src/Applications/rank-object-enumerator.py

```diff
@@ -92,33 +110,33 @@ def compute_arrangement_works(arrangement, alpha, beta, gamma):
     works = {}
     for rank, rank_objects in ranks.items():
         # Compute per-rank loads
-        works[rank] = alpha * compute_load(rank_objects)
+        works[rank] = alpha_c * compute_load(rank_objects)
 
         # Compute communication volumes
-        works[rank] += beta * max(
+        works[rank] += beta_c * max(
             compute_volume(rank_objects, "from"),
             compute_volume(rank_objects, "to"))
 
         # Add constant
-        works[rank] + gamma
+        works[rank] += gamma_c
```

Comment:
@ppebay could you please review this specific line. I changed it, because it doesn't have any impact before

Reply:
Yes this is good, thank you, that was a typo, but we never tested with nonzero `gamma`, so it would have failed. Thanks.